packages/explorer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.1",
+  "private": true,
+  "description": "Explorer is a tool for visually exploring and manipulating the state of worlds",
```

Comment:
are we able to use `"type": "module"` here? that would maybe let us avoid `.mjs` etc everywhere?

Reply:
Works with `type: module`! Updated, no more `mjs`